docs/pages/router/overview.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Expo Router
+description: Welcome to Expo Router
```

Comment:
The description is to short. We use this tag for SEO and open graph. 

```suggestion
description: Welcome to Expo Router - an open-source routing library for React Native applications built with Expo.
```

We can just the first line here. If you have suggestion for a better description, feel free to add that.